metadata.rb

```diff
@@ -19,4 +19,4 @@
 source_url 'https://github.com/chef-cookbooks/passenger_apache2'
 issues_url 'https://github.com/chef-cookbooks/passenger_apache2/issues'
 
-chef_version '>= 12.1'
+chef_version '>= 13.9'
```

Comment:
apache2 7.1 requires chef 13.9, so may as well make that explicit here too.

Reply:
Yep this is a totally safe version of Chef to assume at this point

Reply:
👍